app/logic/updateTrackHours.py

```diff
@@ -0,0 +1,75 @@
+# from app.controllers.admin import admin_bp
+from app.models.eventParticipant import EventParticipant
+from datetime import datetime
+from app.models.user import User
+
+
+def getEventLengthInHours(startTime, endTime, eventDate):
+    """
+    Converts the event length hours into decimal
+    parameters: startTime- start time event (type: time)
+                endTime- end time event (type: time)
+                eventDate- date of the event (type: datetime)
+    """
+    #can only subtract datetime objects, not time objects. So convert time into datetime
+    eventLength = datetime.combine(eventDate, endTime) - datetime.combine(eventDate, startTime)
+    (h, m, s) = str(eventLength).split(':')

```

Comment:
can't you pull these values out of a datetime directly, rather than cast to a string and split?